### PR TITLE
add vars and service for vote in templates

### PR DESCRIPTION
**Pull request by peb-adr**
As the vote-service is getting build into the main repo (see https://github.com/OpenSlides/OpenSlides/pull/6274) these are the equivalent changes for the manage tool.

### CODE REVIEW

File: pkg/config/default-docker-compose.yml [hunk 1]
```diff
@@ -228,6 +238,31 @@ services:
...
+      - auth
+      - redis
+    {{- end }}
+    environment:
+      << : *default-environment
+      MESSAGING: redis
+      AUTH: ticket
+    networks:
+      - datastore-reader
+      - datastore-writer
```
Comment (normanjaeckel) [inline pkg/config/default-docker-compose.yml:258]: @ostcar Does the vote service use the datastore-reader and/or the datastore-writer directly? Or does it get all of its information from the backend?

Please also check networks alltogether. Maybe "frontend" is missing (see e. g. autoupdate and icc)

File: pkg/config/default-config.yml [hunk 2]
```diff
@@ -40,6 +40,16 @@ defaultEnvironment:
   AUTH_HOST: auth
   AUTH_PORT: 9004
 
+  VOTE_HOST: vote
+  VOTE_PORT: 9013
+  VOTE_REDIS_HOST: redis
+  VOTE_REDIS_PORT: 6379
+  VOTE_DATABASE_USER: openslides
```
Comment (normanjaeckel) [inline pkg/config/default-config.yml:47]: Please order the database related variables like in line 30 to 35 (host, port, name, user, password, password_file).

@ostcar Can you please enable reading the db password from a file given as secret? See datastore settings here.

Comment (normanjaeckel) [reply pkg/config/default-config.yml:47]: @peb-adr Please add a new secret for the database password file. The secret may be called `vote_postgres_password`. You also have to update setup/setup.go at the beginning (const section) and around line 123. The randomization is an extra issue.

File: pkg/config/default-docker-compose.yml [hunk 3]
```diff
@@ -23,6 +23,16 @@ x-default-environment: &default-environment
   AUTH_HOST: {{ .AUTH_HOST }}
   AUTH_PORT: {{ .AUTH_PORT }}
 
+  VOTE_HOST: {{ .VOTE_HOST }}
+  VOTE_PORT: {{ .VOTE_PORT }}
+  VOTE_REDIS_HOST: {{ .VOTE_REDIS_HOST }}
+  VOTE_REDIS_PORT: {{ .VOTE_REDIS_PORT }}
+  VOTE_DATABASE_USER: {{ .VOTE_DATABASE_USER }}
+  VOTE_DATABASE_PASSWORD: {{ .VOTE_DATABASE_PASSWORD }}
```
Comment (normanjaeckel) [inline pkg/config/default-docker-compose.yml:31]: Respective changes have also to be done here.

**Review by normanjaeckel — CHANGES_REQUESTED** [review]
Nice work. Thank @peb-adr and @ostcar very much. Please have a look at my comments.

File: pkg/config/default-docker-compose.yml (hunk 1 above)
Comment (ostcar) [reply pkg/config/default-docker-compose.yml:258]: It accesses the datastore-reader but not the datastore-writer.

It needs to be in the same network as the backend, the datastore-reader, the message bus, the Auth service and the autoupdate service.

**Review by normanjaeckel — APPROVED** [review]

**Pull request merged**